core/templates/pages/exploration-player-page/modals/display-hint-modal.component.spec.ts

```diff
@@ -105,8 +108,9 @@ describe('Display hint modal', () => {
     let recordedVoiceovers = new RecordedVoiceovers({});
     let hint = new SubtitledHtml('html', contentId);
     let displayedCard = new StateCard(
-      'test_name', 'content', 'interaction', null, [],
-      recordedVoiceovers, null, contentId, null);
+      'test_name', 'content', 'interaction', {} as Interaction, [],
+      recordedVoiceovers, {} as WrittenTranslations, contentId,
+      {} as AudioTranslationLanguageService);
```

Comment:
Can we not use this typecasting and use something like `new Interaction(...)` etc.

Reply:
test file.